conductr_cli/bndl_utils.py

```diff
@@ -156,6 +157,30 @@ def load_bundle_args_into_conf(config, args, with_defaults):
     if with_defaults and 'annotations' not in config:
         config.put('annotations', ConfigTree(BNDL_DEFAULT_ANNOTATIONS.copy()))
 
+    if args_check_addresses is not None:
+        check_args = []
+        if hasattr(args, 'check_initial_delay') and args.check_initial_delay:
+            check_args += ['--initial-delay', str(args.check_initial_delay)]
+        if hasattr(args, 'check_connection_timeout') and args.check_connection_timeout:
+            check_args += ['--connection-timeout', str(args.check_connection_timeout)]
+        check_args += args_check_addresses
+
+        if 'components' in config:
+            status_component_names = [component for component in config.get('components')
+                                      if component.endswith('-status')]
```

Comment:
What do you think about tightening the test here? My concern is that a user can't end their unrelated component in `-status` even though perhaps it's not a `check` component. Perhaps we can also make sure that `start-command` starts with `check`?